README.md

```diff
@@ -1,6 +1,12 @@
-# HASH
+[github_banner]: https://hash.dev/?utm_medium=organic&utm_source=github_readme_hash-repo_root
+[github_star]: https://github.com/hashintel/hash#
```

Comment:
```suggestion
[github_star]: https://github.com/hashintel/hash
```

? (here and elsewhere)

Reply:
I don't want to reload the pages in all instance, but to jump the user up to the top of the page where the star button is (if they're on that page already). This achieves that.